frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -892,20 +892,14 @@ user_pill.get_user_ids = function () {
         assert.deepEqual(returned, reference);
     }
 
-    var all_items = [
-        {
-            special_item_text: 'all (Notify everyone)',
-            email: 'all',
-            pm_recipient_count: Infinity,
-            full_name: 'all',
-        },
-        {
-            special_item_text: 'everyone (Notify everyone)',
-            email: 'everyone',
+    var all_items = _.map(['all', 'everyone', 'stream'], function (mention) {
+        return {
+            special_item_text: mention +" (Notify everyone)",
```

Comment:
This will merge-conflict with 721b4e83733b0e5f569737cb1469deb1526b9af8; can you resolve those conflicts?